### PR TITLE
Sync with the EN page: (Composite components) Remove 自動テスト section

### PR DESCRIPTION
**Pull request by takuyakikuchi**
## Linked issue

No issue.

## Goal

Sync [Composite components](https://storybook.js.org/tutorials/intro-to-storybook/react/ja/composite-component/) with the English tutorial page.

## Changes

Removed 自動テスト section which doesn't exist on the English page.

I tried 自動テスト section but, there was no instruction about installing `'@storybook/testing-react'` and didn't work with what was described there.

It seems like the whole section was depended on the older version of the tutorial? 🤔 
> 前の章で Storyshots を使用してストーリーのスナップショットテストを行う方法を学びました。
For example, there was no snap shot test so far at this point.

## UI

None.

## Other notes

None.

### CODE REVIEW

File: content/intro-to-storybook/react/ja/composite-component.md [hunk 1]
```diff
@@ -221,54 +221,3 @@ export default function TaskList({ loading, tasks, onPinTask, onArchiveTask }) {
...
-
-![TaskList のテストランナー](/intro-to-storybook/tasklist-testrunner.png)
-
-単体テストで `WithPinnedTasks` ストーリーを再利用出来ていることに注目してください。このように、多様な方法で既存のリソースを活用していくことができます。
-
-ただし、このテストは非常に脆いことにも留意してください。プロジェクトが成熟し、`Task` の実装が変わっていく (たとえば、別のクラス名に変更されたり、`input` 要素ではなく `textarea` に変更されたりする) と、テストが失敗し、更新が必要となる可能性があります。これは必ずしも問題とならないかもしれませんが、UI の単体テストを使用する際の注意点です。UI の単体テストはメンテナンスが難しいのです。可能な限り手動のテストや、スナップショットテスト、視覚的なリグレッションテスト ([テストの章](/intro-to-storybook/react/ja/test/)を参照してください) に頼るようにしてください。
-
-<div class="aside">
-💡 Git へのコミットを忘れずに行ってください！
-</div>
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/ja/composite-component.md:274]: @takuyakikuchi if you're ok with it. Could you re-add the aside that you removed as it's a bit important, we want the readers to keep a git history of their commits throughout the tutorial.

**Comment by jonniebigodes** [comment]
@takuyakikuchi, thanks for taking the time to update the tutorial. We're truly thankful for the time and effort you've put into this pull request. One small item and this should be good to go. Let me know once you've addressed it and I'll gladly merge the pull request.

Hope you have a great week.

Stay safe

File: content/intro-to-storybook/react/ja/composite-component.md (hunk 1 above)
Comment (takuyakikuchi) [reply content/intro-to-storybook/react/ja/composite-component.md]: Yap, you are right! Thank you for catching this!

I have put it back with the following commit!
https://github.com/chromaui/learnstorybook.com/pull/666/commits/d9ad2c7b6b27ae568a14fd14463b6ccbafbb30ba

**Comment by jonniebigodes** [comment]
@takuyakikuchi, all is good on my end. Thanks for addressing the feedback so promptly. Appreciate it 🙏 ! Going to merge this.

Hope you have a great day!

Stay safe

**Pull request merged**